# List of datasets to be used

## 2D datasets

- HE datasets from the TCGA (HE) to be correlated with molecular data: https://portal.gdc.cancer.gov/
- Ebrains datasets (more than 120 types of brain tumors with HE data): https://search.kg.ebrains.eu/instances/Dataset/8fc108ab-e2b4-406f-8999-60269dc1f994
- GTEx HE and RNAseq: https://gtexportal.org/home/
- Multi-modal cell segmentation dataset: https://neurips22-cellseg.grand-challenge.org/

## 2D and 3D microglia morphology
- Paper with some ideas on different analyses to perform on microglia morphology: https://pure.rug.nl/ws/portalfiles/portal/783390917/Glia_-_2023_-_Weering.pdf
- General review on the microglia morphology definition: https://pubmed.ncbi.nlm.nih.gov/36327895/
- Cell motility and morphology evaluation method: https://www.molbiolcell.org/doi/10.1091/mbc.E21-11-0585#_i24 and https://www.eneuro.org/content/eneuro/11/7/ENEURO.0014-24.2024.full.pdf (method https://github.com/ciernialab/MicrogliaMorphology), https://pubmed.ncbi.nlm.nih.gov/29288586/ and https://pmc.ncbi.nlm.nih.gov/articles/PMC6325541/
  1) Dataset: https://osf.io/8t5c2/
  2) In vivo SPIM and confocal images of microglia: https://www.ebi.ac.uk/biostudies/BioImages/studies/S-BIAD564?query=microglia
  3) Comparative dynamics of microglialand glioma cell motility at the infiltrative margin of brain tumours: https://zenodo.org/records/4988966
- Different 3D brain cells datasets: https://www.microns-explorer.org/cortical-mm3#segmentation-meshes
- Multi-modal brain cells datasets: https://knowledge.brain-map.org/data?limit=50&offset=0&sort=species.name~ASC
- 2D vs 3D mophology microglia analysis comparison: https://www.biorxiv.org/content/10.1101/2024.09.30.615842v1.full.pdf (data potentially available here https://www.biorxiv.org/content/10.1101/2023.08.28.555135v1.full or https://pubmed.ncbi.nlm.nih.gov/37689276/)

## 3D datasets

- 3D cell segmentation
   1) (CHAMMI dataset): https://zenodo.org/records/7988357
   2) method https://github.com/chaudatascience/channel_adaptive_models?tab=readme-ov-file

- 3D neuronal networks from three-dimensional electron microscopy (3D-EM) 
  1) Paper (dataset inside): https://www.nature.com/articles/s41592-024-02226-5
  2) Paper (dataset inside): https://www.nature.com/articles/s41592-022-01711-z
      
- 3D different types of cells
   1) ovules: https://osf.io/uzq3w/
   2) ATAS: https://www.repository.cam.ac.uk/handle/1810/262530
   3) Arabidopsis: https://osf.io/fzr56/
   4) using Cellstich https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05608-2
      
- 3D brain MRI + histology
    1) https://rdr.ucl.ac.uk/articles/dataset/Registered_histology_MRI_and_manual_annotations_of_over_300_brain_regions_in_5_human_hemispheres_data_from_ERC_Starting_Grant_677697_BUNGEE-TOOLS_/24243835
    2) method https://github.com/acasamitjana/ERC_reconstruction
    3) dataset: https://medical-epigenomics.org/papers/GBMatch/#data_downloads (WSI, MRI, methylation and clinical data)

- 3D brain MRI + ultrasound
   1) Dataset REMIND: https://www.cancerimagingarchive.net/collection/remind/
       
- 3D spatial transcriptomic
    1) data: https://www.bgiocean.com/vt3d_example/
    2) article https://www.sciencedirect.com/science/article/pii/S1673852723000772
       
- 3D spatial transcriptomics
    1) data: https://rajewsky-lab.github.io/openst/examples/getting_started/
    2) method https://github.com/rajewsky-lab/openst?tab=readme-ov-file
    3) article https://www.sciencedirect.com/science/article/pii/S0092867424006366?dgcid=coauthor
       
- 3D histology
    1) method: https://github.com/WeisiX/ITAS3D (data potentially available)
    2) method: https://www.cell.com/cell/abstract/S0092-8674(24)00351-9?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867424003519%3Fshowall%3Dtrue (data potentially available)
    3) method: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10500590/ (data potentially available)
    4) 3D melanoma https://www.ejcancer.com/article/S0959-8049(21)01126-6/fulltext (data potentially available)
  
- 3d  hierarchical phase-contrast tomography (full organ at cell resolution)
    1) data: brain and lung high-resolution https://human-organ-atlas.esrf.eu/ 
    1) article: https://www.nature.com/articles/s41592-021-01317-x
       
- 3D histology (prostate cancer)
    1) data: https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=145754446
    2) article https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10926776/

- 3D histology (different cancers)
    1) data: https://data.humantumoratlas.org/ (different types of cancer, several technologies)
    2) https://github.com/labsyspharm/CRC_atlas_2022 (details inside)
    3) https://www.synapse.org/#!Synapse:syn18434611/wiki/597418
       
- 3D retinal vessels
    1) data https://huggingface.co/datasets/Hemaxi/3DVesselSegmentation
    2) article https://www.ahajournals.org/doi/10.1161/ATVBAHA.124.320672
       
- 3D virtual histopathology of cardiac tissue from Covid-19 patients based on phase-contrast X-ray tomography
    1) data: https://zenodo.org/records/5658380
    2) article: https://elifesciences.org/articles/71359
       
- 3D image mass cytometry
    1) data: https://portal.hubmapconsortium.org/ (inside there are 2D spatial transcriptomics, 2D spatial proteomics and other types of data).
       
- 3D spatial metabolomics
    1) data: https://zenodo.org/records/4087670
    2) article: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7840140/
